apps/tailwind-components/components/button/Text.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
```

Comment:
how to call this component? And I notice the css doesn't work on all themes properly. Need help there.